test/integration/targets/ansible-config/tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+- name: test ansible-config for valid output and no dupes
+  block:
+    - name: Create temporary file
```

Comment:
You can remove the block and manual management of a tempdir by adding the following `meta/main.yml` file to the test role and using the `remote_tmp_dir` var in these tasks.

```yaml
dependencies:
- setup_remote_tmp_dir
```